docs/metrics.md

```diff
@@ -0,0 +1,67 @@
+# Firecracker Metrics Configuration
+
+For the metrics capability, Firecracker uses a generic Metrics system.
```

Comment:
I don't think generic is the appropriate word here. What do we want to say by generic?